backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -664,3 +670,285 @@ async def get_facility_QR_for_autologin(
     except Exception as e:
         return {"status": "error", "message": str(e)}
 
+@router.post('/amcConfigurationTemplate',
+            summary='Generate AMC configuration ingestion template',
+            response_description="Returns Excel template with facility asset metadata for AMC configurations")
+async def get_amc_configuration_template(
+        background_tasks: BackgroundTasks,
+        payload: dict = Body(..., description="Payload containing RequestInfo, boundary_data")
```

Ruff (0.14.5)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)